front-end-101/final_project_plan/about.html

```diff
@@ -27,7 +27,7 @@
             <div>
                 <h1>Меня зовут Ерик. 32 года. На эти 6 месяцев обучения у меня есть несколько целей.</h1>
             </div>
-            <div class="about">
+            <div class="about" id="about">
```

Comment:
class about is not needed here if you are using ID and applying similar css properties